metaflow/plugins/project_decorator.py

```diff
@@ -25,6 +25,50 @@ class ProjectDecorator(FlowDecorator):
         Project name. Make sure that the name is unique amongst all
         projects that use the same production scheduler. The name may
         contain only lowercase alphanumeric characters and underscores.
+
+    MF Add To Current
+    -----------------
+    project_name -> str
```

Comment:
add multiple properties. Blank line between properties is optional I think but makes things clearer.